app/cmd/root.go

```diff
@@ -34,34 +30,33 @@ func init() {
 		config.P2P.ConnKeepAlive, "Network connection keep alive")
 	RootCmd.PersistentFlags().Int8Var(&config.P2P.NetworkID, "network-id",
 		config.P2P.NetworkID, "NetworkID to run on (0 - mainnet, 1 - testnet)")
-	RootCmd.PersistentFlags().DurationVar(&config.P2P.ResponseTimeout, "response-timeout",
-		config.P2P.ResponseTimeout, "Timeout for waiting on resposne message")
 	RootCmd.PersistentFlags().StringVar(&config.P2P.NodeID, "node-id",
 		config.P2P.NodeID, "Load node data by id (pub key) from local store")
 	RootCmd.PersistentFlags().BoolVar(&config.P2P.NewNode, "new-node",
 		config.P2P.NewNode, "Load node data by id (pub key) from local store")
     RootCmd.PersistentFlags().IntVar(&config.P2P.BufferSize, "buffer-size",
 		config.P2P.BufferSize, "Size of the messages handler's buffer")
-	RootCmd.PersistentFlags().BoolVar(&config.P2P.SwarmConfig.Gossip, "gossip",
-		config.P2P.SwarmConfig.Gossip, "should we start a gossiping node?")
-	RootCmd.PersistentFlags().BoolVar(&config.P2P.SwarmConfig.Bootstrap, "bootstrap",
-		config.P2P.SwarmConfig.Bootstrap, "Bootstrap the swarm")
-	RootCmd.PersistentFlags().IntVar(&config.P2P.SwarmConfig.RoutingTableBucketSize, "bucketsize",
-		config.P2P.SwarmConfig.RoutingTableBucketSize, "The rounding table bucket size")
-	RootCmd.PersistentFlags().IntVar(&config.P2P.SwarmConfig.RoutingTableAlpha, "alpha",
-		config.P2P.SwarmConfig.RoutingTableAlpha, "The rounding table Alpha")
-	RootCmd.PersistentFlags().IntVar(&config.P2P.SwarmConfig.RandomConnections, "randcon",
-		config.P2P.SwarmConfig.RoutingTableAlpha, "Number of random connections")
-	RootCmd.PersistentFlags().StringSliceVar(&config.P2P.SwarmConfig.BootstrapNodes, "bootnodes",
-		config.P2P.SwarmConfig.BootstrapNodes, "Number of random connections")
-	RootCmd.PersistentFlags().DurationVar(&config.P2P.TimeConfig.MaxAllowedDrift, "max-allowed-time-drift",
-		config.P2P.TimeConfig.MaxAllowedDrift, "When to close the app until user resolves time sync problems")
-	RootCmd.PersistentFlags().IntVar(&config.P2P.TimeConfig.NtpQueries, "ntp-queries",
-		config.P2P.TimeConfig.NtpQueries, "Number of ntp queries to do")
-	RootCmd.PersistentFlags().DurationVar(&config.P2P.TimeConfig.DefaultTimeoutLatency, "default-timeout-latency",
-		config.P2P.TimeConfig.DefaultTimeoutLatency, "Default timeout to ntp query")
-	RootCmd.PersistentFlags().DurationVar(&config.P2P.TimeConfig.RefreshNtpInterval, "refresh-ntp-interval",
-		config.P2P.TimeConfig.RefreshNtpInterval, "Refresh intervals to ntp")
+	RootCmd.PersistentFlags().BoolVar(&config.P2P.DHTConfig.Gossip, "gossip",
```

Comment:
Let's put gossip under `P2P`
also let's change `DHTConfig` to `DiscoveryConfig`

Reply:
sorry for that 😅

Reply:
> Let's put `Gossip` under `P2P`

Do you mean put `Gossip` under P2P directly but not this `DHTConfig`?
Can we delete it? I found no code read this `Gossip` config.

Reply:
> also let's change `DHTConfig` to `DiscoveryConfig` (sorry for that 😅 )

No problem 😃. Fixed in 3d0c92b1bd1dae9d28b93345d12f6bd3b726455f.

Reply:
> if you have any questions feel free to DM me on gitter or just ask here.

No problem.

Reply:
@y0sher Fixed. Please review it.